### PR TITLE
Adding Continuous Pub topic - erikre

### PR DESCRIPTION
**Pull request by Erikre**
Please review - @rachelappel @Rick-Anderson @danroth27

### CODE REVIEW

**Comment by danroth27** [comment]
@davidebbo Could we get someone form the Kudu team to also look this over?

**Comment by davidebbo** [comment]
Did a quick read through, and it seems fine. In the end when it comes to Azure/Kudu there isn't anything is there that is different from pushing a non-DNX site: you enable git and you push your repo.

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 1]
```diff
@@ -1,8 +1,246 @@
...
+
+	.. image:: azure-continuous-deployment/_static/10-team-explorer.png
+
+2. In **Team Explorer**, select the **Home** (home icon) > **Settings** > **Repository Settings**.
+
+3. In the **Remotes** section of the **Repository Settings** select **Add**. The **Add Remote** dialog box will be displayed.
+
+4. Set the **Name** of the remote to **Azure-SampleApp**.
+
+5. Set the value for **Fetch** to the **Git URL** that you copied from Azure earlier in this tutorial. Note that this is the URL ends with **.git**.
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: > this is the URL ends with 

this is the URL **that** ends with

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 2]
```diff
@@ -1,8 +1,246 @@
...
+10. Copy the **GIT URL** value for later use in the tutorial.
+
+	.. image:: azure-continuous-deployment/_static/09-azure-giturl.png
+
+Publish your web app to Azure App Service
+-----------------------------------------
+
+In this section, you will create a local Git repository using Visual Studio and push from that repository to Azure to deploy your web app. The steps involved include the following:
+
+	* Add the remote git reference to deploy your local repository to Azure.
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: > Add a remote git reference

In Git I believe the common terminology is a "remote repository" or just a "remote" for short. See http://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes.

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 3]
```diff
@@ -1,8 +1,246 @@
...
+	
+|	remote: Finished successfully.
+|	remote: Running post deployment command(s)...
+|	remote: Deployment successful.
+|	To https://username@samplewebappdemo01.scm.azurewebsites.net:443/SampleWebAppDemo01.git
+|	* [new branch]      master -> master
+|	Branch master set up to track remote branch master from Azure-SampleApp.
+|
+
+.. note:: If you need to collaborate on a project, you should consider pushing to GitHub in between pushing to Azure.
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: Make "GitHub" a link to https://github.com

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 4]
```diff
@@ -1,8 +1,246 @@
...
+2. After reviewing the running Web app, close the browser and click the "Stop Debugging" icon in the toolbar of Visual Studio to stop the app.
+
+.. note:: To review the ASP.NET 5 project, see the "Review the project" section of :doc:`/tutorials/your-first-aspnet-application`.
+
+Create a web app in the Azure Portal
+------------------------------------
+
+The following steps will guide you through creating a web app in the Azure Portal.
+
+1. Log in to the `Azure Portal <https://portal.azure.com>`_.
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: What if the reader doesn't have an Azure account? Can we point them to create a free account?

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 5]
```diff
@@ -1,8 +1,246 @@
...
+
+View your updated web app by selecting **Browse** from the web app blade in the Azure Portal or by opening a browser and entering the URL for your web app. For example:
+
+	``http://SampleWebAppDemo.azurewebsites.net``
+
+Additional Resources
+--------------------
+- :doc:`/publishing/index`. 
+- :doc:`/conceptual-overview/understanding-aspnet5-apps`.
+- `ASP.NET 5 Fundamentals <https://docs.asp.net/en/latest/fundamentals/index.html>`_
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: This should be a doc reference like the previous links:
```
:doc:`/fundamentals/index`
```

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 6]
```diff
@@ -1,8 +1,246 @@
...
+		``git push -u Azure-SampleApp master``
+
+View the updated web app in Azure
+---------------------------------
+
+View your updated web app by selecting **Browse** from the web app blade in the Azure Portal or by opening a browser and entering the URL for your web app. For example:
+
+	``http://SampleWebAppDemo.azurewebsites.net``
+
+Additional Resources
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: Are there other Azure topics that give more info on Kudu that we can't point folks to?

**Comment by danroth27** [comment]
Nice topic! Just a few quick comments. 
:watch:

**Comment by Erikre** [comment]
Updated have been made. @danroth27 can I get a shipit?

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 7]
```diff
@@ -1,8 +1,249 @@
...
+Publish your web app to Azure App Service
+-----------------------------------------
+
+In this section, you will create a local Git repository using Visual Studio and push from that repository to Azure to deploy your web app. The steps involved include the following:
+
+	* Add the remote repository setting using your GIT URL value, so you can deploy your local repository to Azure.
+	
+	* Commit your project changes.
+	
+	* Push your project changes from your local repository to Azure using the remote reference.
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: > Push your project changes from your local repository to Azure using the remote reference

Push your project changes from your local repository to your remote repository on Azure

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 8]
```diff
@@ -1,8 +1,249 @@
...
+View the updated web app in Azure
+---------------------------------
+
+View your updated web app by selecting **Browse** from the web app blade in the Azure Portal or by opening a browser and entering the URL for your web app. For example:
+
+	``http://SampleWebAppDemo.azurewebsites.net``
+
+Additional Resources
+--------------------
+- ASP.NET 5 :doc:`/publishing/index`
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: This is going to result in some odd formatting where just the link is italicized. Just override the entire tile:
```
:doc:`ASP.NET 5 Publishing </publishing/index>`
```

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 9]
```diff
@@ -1,8 +1,249 @@
...
+- `Project Kudu <https://github.com/projectkudu/kudu/wiki>`_
+- :doc:`/conceptual-overview/understanding-aspnet5-apps`
+- ASP.NET 5 :doc:`/fundamentals/index`
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: This is going to result in some odd formatting where just the link is italicized. Just override the entire tile:

```
:doc:`ASP.NET 5 Fundamentals </fundamentals/index>`
```

File: aspnet/publishing/azure-continuous-deployment.rst [hunk 10]
```diff
@@ -1,8 +1,249 @@
...
+
+4. Set the **Name** of the remote to **Azure-SampleApp**.
+
+5. Set the value for **Fetch** to the **Git URL** that you copied from Azure earlier in this tutorial. Note that this is the URL that ends with **.git**.
+
+	.. image:: azure-continuous-deployment/_static/11-add-remote.png
+	
+	|
+	
+	.. note:: As an alternative, you can add the remote reference from the **Command Window** by opening the **Command Window**, changing to your project directory, and entering the command. For example:
```
Comment (danroth27) [inline aspnet/publishing/azure-continuous-deployment.rst]: > you can add the remote reference

you can specify the remote repository

**Comment by danroth27** [comment]
A couple more comments :watch:

**Comment by Erikre** [comment]
New updates. @danroth27 need shipit.

**Comment by danroth27** [comment]
:shipit:

**Comment by Erikre** [comment]
d730b9c5d3b51c6efb26323ab72d3a09b3e5c2e2

**Pull request closed without merging**